airbyte-integrations/connectors-performance/destination-harness/src/main/java/io/airbyte/integrations/destination_performance/Main.java

```diff
@@ -86,13 +96,37 @@ public static void main(final String[] args) {
     System.exit(0);
   }
 
-  static JsonNode getCatalog(final String dataset, final String connector) throws IOException {
-    final ObjectMapper objectMapper = new ObjectMapper();
-    final String catalogFilename = "catalogs/%s/%s_catalog.json".formatted(connector, dataset);
-    final InputStream is = Thread.currentThread().getContextClassLoader().getResourceAsStream(catalogFilename);
-    return objectMapper.readTree(is);
+  /**
+   * Duplicate the streams in the catalog to emulate parallel streams
+   *
+   * @param root the catalog
+   * @param duplicateFactor the number of times to duplicate each stream
+   */
+  private static void duplicateStreams(final JsonNode root, final int duplicateFactor) {
+    try {
+      final ObjectNode streamObject = (ObjectNode) root.path("streams").get(0);
+      // Since we already have one stream, we only need to duplicate the remaining streams
+      for (int i = 1; i < duplicateFactor; i++) {
+        final ObjectNode newStream = streamObject.deepCopy();
+        final String streamName = newStream.path("stream").path("name").asText();
+        ((ObjectNode) newStream.get("stream")).put("name", streamName + i);
```

Comment:
The bottleneck catalog is already names bottleneck_stream1 IIRC.
Is the 1 going to play nice with duplication?

Reply:
You're right, it will still work you'll just get something like `bottleneck_stream11` instead of `bottleneck_stream1`